examples/networking/router_route/base-compute-instance.tf

```diff
@@ -0,0 +1,16 @@
+resource "vkcs_compute_instance" "basic-compute-tf-example" {
```

Comment:
base-compute-instance -> base-compute-gateway
basic-compute-tf-example -> gateway

for name: gateway-tf-example

Reply:
done